README.md

```diff
@@ -392,6 +392,8 @@ And we're back! Use this repo to share and keep track of software, tech, CS, PM,
 | [r2c](https://jobs.lever.co/returntocorp/82c10097-a669-417a-9434-66a98a8bca1f) | San Francisco, CA; or Remote | Software Engineer - Internship, Application Development |
 | [Excelitas](https://www.excelitas.com/job/2022-10/software-engineering-intern) | Billerica, MA | Software Engineering Intern (No Visa Sponsorship) |
 | [Teradata](https://careers.teradata.com/jobs/215587/cloud-software-engineering-intern) | Remote | Cloud Software Engineering Intern |
+| [klue](https://jobs.lever.co/klue/b88e45c3-c067-410b-a7d3-ca23142b9bef) | Vancouver, BC | Data Analyst Intern |
+| [LINK](https://careers-linkindustrialfields.icims.com/jobs/search?ss=1&searchPositionType=18701) | Multiple Locations | Data Analyst Intern] |
```

Comment:
Since you're mentioning the data analyst intern, have your link point straight to the application, e.g., `https://careers-linkindustrialfields.icims.com/jobs/4602/data-science-intern%2c-research-%26-analytics-2023/job`. 

Or, if your intent was to link multiple internships (since I saw DS, CE, Data Analyst), mention those roles explicitly and (preferably) link them individually through the description.

In other words, for the entry of LINK, you can just use (and ensure it renders correctly):
```
| [LINK](https://careers-linkindustrialfields.icims.com/jobs/search?ss=1&searchPositionType=18701) | Multiple Locations | [Data Science Intern](https://careers-linkindustrialfields.icims.com/jobs/4602/data-science-intern%2c-research-%26-analytics-2023/job?in_iframe=1), [Computer Engineering Intern](https://careers-linkindustrialfields.icims.com/jobs/4459/computer-engineering-summer-intern-2023/job?in_iframe=1), [Data Analyst Intern](https://careers-linkindustrialfields.icims.com/jobs/4450/data-analyst-intern-2023/job?in_iframe=1) |
```